examples/with-firebase-authentication/utils/auth/mapUserData.js

```diff
@@ -0,0 +1,8 @@
+export const mapUserData = user => {
+    const { uid, email, xa } = user
```

Comment:
@joejknowles Do you know where this `xa` property is coming from? Doesn't seem to be documented in the Firebase SDK and I feel like I'm losing my mind.

Reply:
I don't, this was just from the existing code

Reply:
I mean, I think it was on the user object from Firebase, but I didn't find any documentation or anything.

Reply:
@vriad this was in the coffee before actually, if it helps:
`
// xa is the access token, which can be retrieved through
// firebase.auth().currentUser.getIdToken()
`

Reply:
Interesting. thank you 👍

Reply:
@joejknowles is correct, `xa`is the access token from Firebase 👍